scripts/localnet

```diff
@@ -37,6 +37,7 @@ bob() {
         --tmp \
         --bob \
         --port 30334 \
+        --unsafe-force-node-key-generation \
```

Comment:
Will the update also affect how we run the network? Should we specify this flag for our nodes?

Reply:
It allows node to generate on run any _node key_ when the expected one is not found.
This _cli key_ can be omitted if the _node key_ is generated explicitly at some step. So I think, no, we don't need this in our nodes.

Reply:
Do we need it if we didn’t set the node key with a flag (explicitly)?

Reply:
I think yes.
https://github.com/paritytech/polkadot-sdk/blob/8c8edacf8942298c3807a2e192860da9e7e4996a/substrate/client/cli/src/params/node_key_params.rs#L118

Reply:
Then we need to update our deploy scripts as well with either setting the ids or adding this flag, as in this case the change will break them. Could you open an issue and mark it as blocking for this PR? Needs a little investigation how much it’s “unsafe” to use to this flag and whether we need to generate the ids.